cmd/ethrex_l2/src/commands/stack.rs

```diff
@@ -276,6 +293,64 @@ impl Command {
                     current_block += U256::one();
                 }
             }
+            Command::Reconstruct {
```

Comment:
Nit: the error handling is a bit inconsistent. We should probably choose between `panic` and derivatives everywhere or nice errors with `eyre` everywhere. I believe both are pretty much the same amount of work.

Reply:
Moved to `expect` for this command on [c031265](https://github.com/lambdaclass/ethrex/pull/2204/commits/c03126557a8f58951bb64503bdc37a14417508c0). We can open a new PR to standarize this